src/js/hooks/useWizard.js

```diff
@@ -59,11 +61,11 @@ const useWizard = ({ initialKey, steps }) => {
     navigateToStep(steps[lastIdx]?.key);
   };
 
-  const next = () => {
+  const next = (params = {}) => {
```

Comment:
add a comment above explaining why we could need the params (e.g. "params might be needed to join some query params to the URL while switching the step")